src/saving_tools.jl

```diff
@@ -19,24 +19,35 @@ julia> current_commit()
 
 julia> current_commit(path_to_dirty_repo)
 "3bf684c6a115e3dce484b7f200b66d3ced8b0832_dirty"
+
+julia> current_commit(another_path_with_annotated_git_tags)
+"v1.2.3-g7364ab"
 ```
 """
 function current_commit(gitpath = projectdir())
     # Here we test if the gitpath is a git repository.
     try
         repo = LibGit2.GitRepo(gitpath)
     catch er
-        @warn "The current project directory is not a Git repository, "*
```

Comment:
Here `*` is necessary. If you remove it the string on the line below is not chained to the above and only the first line is displayed as a warning.

Reply:
Yep! That was a typo... thanks!